satpy/readers/yaml_reader.py

```diff
@@ -475,11 +477,12 @@ def new_filehandlers_for_filetype(self, filetype_info, filenames):
             # to reduce the number of files to open
             filename_iter = self.filter_filenames_by_info(filename_iter)
         filehandler_iter = self.new_filehandler_instances(filetype_info,
-                                                          filename_iter)
+                                                          filename_iter,
+                                                          reader_kwargs=reader_kwargs)
         filtered_iter = self.filter_fh_by_metadata(filehandler_iter)
         return list(filtered_iter)
 
-    def create_filehandlers(self, filenames):
+    def create_filehandlers(self, filenames, reader_kwargs=None):
```

Comment:
This should probably be called something else for future flexibility. Something like `fh_kwargs` or even take a `**kwargs` (@pnuu @mraspaud will disagree). Although the kwargs are coming from the readers kwargs currently, this may be separated out in the future.

Reply:
I second `fh_kwargs`

Reply:
ok done